runtime/compiler/env/J9VMMethodEnv.cpp

```diff
@@ -74,3 +77,39 @@ J9::VMMethodEnv::bytecodeSize(TR_OpaqueMethodBlock *method)
                      J9_BYTECODE_START_FROM_ROM_METHOD(romMethod));
    }
 
+
+void
+J9::VMMethodEnv::tokenizeSignature(
+      const char * methodSignature,
+      const char * &methodClass,
+      uint32_t     &methodClassLen,
+      const char * &methodName,
+      uint32_t     &methodNameLen,
+      const char * &typeSignature,
+      uint32_t     &typeSignatureLen)
+   {
+
+   if (methodSignature[0] == '/') // omr method pattern
```

Comment:
Why does OpenJ9 need to handle the omr method pattern?

Reply:
_What_ is an OMR method pattern, anyway?